hypha/apply/funds/templates/funds/includes/co-applicant-block.html

```diff
@@ -0,0 +1,60 @@
+{% load i18n heroicons %}
+
+<div class="sidebar__inner sidebar__inner--actions" data-testid="sidebar-primary-actions">
+    <details {% if user.is_applicant %} open {% endif %}>
+        <summary class="flex gap-4 justify-between items-center">
+            <p class="flex items-center text-lg font-bold">
+                {% trans "Co-applicants" %}
+                {% heroicon_solid "chevron-down" size="16" class="w-4 h-4 ms-2" %}
+            </p>
+            <button
+                class="flex items-center py-1 pr-4 pl-2 font-bold rounded-sm -me-2 {% if object.co_applicant_invites.count >= invite_max_limit %} text-blue-200 {% else %}transition-colors cursor-pointer text-dark-blue hover:bg-slate-200{% endif %}"
+                hx-get="{% url 'apply:submissions:invite_co_applicant' pk=object.id %}"
+                hx-target="#htmx-modal"
+                {% if object.co_applicant_invites.count >= invite_max_limit %}disabled title='{% trans "Max limit reached" %}'{% endif %}
+                role="button"
+                aria-label="{% trans "Invite CoApplicant" %}"
+            >
+                {% heroicon_micro "plus" class="inline align-middle me-1" stroke_width=5 aria_hidden=true %}
+                {% trans "Invite" %}
+            </button>
+        </summary>
+        {% if co_applicants %}
+            <div class="flex flex-col gap-2 justify-between">
+                {% for invite in co_applicants %}
+
+                    <div>
+                        <a
+                            class="font-bold line-clamp-2"
+                            href="{% url 'apply:submissions:edit_co_applicant' invite_pk=invite.id %}"
+                            hx-get="{% url 'apply:submissions:edit_co_applicant' invite_pk=invite.id %}"
+                            hx-target="#htmx-modal"
+                        >
+                            <div class="flex justify-between">
+                                {% if invite.status == "accepted" %}
+                                    {{ invite.co_applicant.user }}
+                                {% else %}
+                                    {{ invite.invited_user_email }}
+                                {% endif %}
+
+                                {% heroicon_solid "pencil" class="inline align-middle me-1" width=16 height=16 stroke_width=2 aria_hidden=true %}
+
+                            </div>
+                        </a>
+                        {% if invite.status == "accepted" %}
+                            <div class="text-xs text-fg-muted" >{{ invite.co_applicant.get_role_display }}</div>
+                        {% else %}
+                            <div class="text-xs text-fg-muted" >{{ invite.get_status_display }}</div>
+                        {% endif %}
+                    </div>
+
+                {% endfor %}
+            </div>
+        {% else %}
+            <div>
+                {% trans "No co-applicant yet" %}
```

Comment:
```suggestion
                {% trans "No co-applicants yet." %}
```
To be consistent with the other blocks

Reply:
updated